packages/camera/camera_web/example/integration_test/camera_service_test.dart

```diff
@@ -89,8 +89,10 @@ void main() {
 
         await cameraService.getMediaStreamForOptions(options);
 
-        expect(capturedConstraints?.video.dartify(), options.video.toJson());
-        expect(capturedConstraints?.audio.dartify(), options.audio.toJson());
+        expect(capturedConstraints?.video.dartify(),
+            equals(options.video.toMediaStreamConstraints().dartify()));
```

Comment:
nit: I much prefer using trailing commas such as

```dart
expect(
  capturedConstraints?.video.dartify(),
  equals(options.video.toMediaStreamConstraints().dartify()),
);
expect(
  capturedConstraints?.audio.dartify(),
  equals(options.audio.toMediaStreamConstraints().dartify()),
);
```

But this is definitely non-blocking

Reply:
Yep, good change! I forgot to reformat those, thanks for picking it up! Let's land this!